lib/cloud-metadata/callback-coordination.js

```diff
@@ -40,7 +40,7 @@ class CallbackCoordination extends EventEmitter {
       this.timeout = setTimeout(() => {
         if (!this.done) {
           this.complete()
-          this.logger.error('metadata requests did not finish, possible deadlock')
+          this.logger.error('metadata requests timed out, using default values instead')
```

Comment:
```suggestion
          this.logger.warn('cloud metadata requests timed out, using default values instead')
```

What do you think about making this a warning? I don't think a user typically needs to act on this.